ambry-router/src/test/java/com.github.ambry.router/GetBlobInfoOperationTest.java

```diff
@@ -149,7 +154,7 @@ public void onCompletion(BlobInfo result, Exception exception) {
     // test a good case
     GetBlobInfoOperation op =
         new GetBlobInfoOperation(routerConfig, routerMetrics, mockClusterMap, responseHandler, blobIdStr,
-            operationFuture, operationCallback, time);
+            operationFuture, operationCallback, operationCompleteCallback, time);
```

Comment:
because this operation is not taken to completion for the decrement to happen (unlike the rest of the operations).